english/quit-your-job/quit-your-job.md

```diff
@@ -0,0 +1,5 @@
+You suddenly decide that coding and enjoying your life are far mor important than saving money. You decide that you are going to take the leap and quit your job in December 2015. As a result you must decide to 
+
+[Move to Brazil](../Brazil.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.